zigbeephilipshue/README.md

```diff
@@ -15,7 +15,7 @@ The [Hue outdoor sensor](https://www.philips-hue.com/en-us/p/hue-outdoor-sensor/
 
 ### Hue dimmer switch
 
-The [Hue dimmer switch](https://www.philips-hue.com/en-us/p/hue-dimmer-switch--latest-model-/046677562779) (including the older model version) is fully supported.
+The [Hue dimmer switch](https://www.philips-hue.com/en-us/p/hue-dimmer-switch--latest-model-/046677562779  https://www.philips-hue.com/en-us/p/hue-dimmer-switch--latest-model-/046677562779) (including the older model version) is fully supported.
```

Comment:
I think a MD link tag can't have 2 links. Besides, isn't it the same link twice? Looks like a copy/paste mistake maybe?